api/web3server.go

```diff
@@ -477,15 +481,39 @@ func (svr *web3Handler) sendRawTransaction(in *gjson.Result) (interface{}, error
 	if err != nil {
 		return nil, err
 	}
-	elp, err := svr.ethTxToEnvelope(tx)
-	if err != nil {
-		return nil, err
-	}
-	req := &iotextypes.Action{
-		Core:         elp.Proto(),
-		SenderPubKey: pubkey.Bytes(),
-		Signature:    sig,
-		Encoding:     encoding,
+	if g := cs.Genesis(); g.IsToBeEnabled(cs.TipHeight()) {
+		if strings.HasPrefix(rawString, "0x") || strings.HasPrefix(rawString, "0X") {
+			rawString = rawString[2:]
+		}
+		rawBytes, _ := hex.DecodeString(rawString)
+		req = &iotextypes.Action{
+			Core: &iotextypes.ActionCore{
+				Version:  version.ProtocolVersion,
+				Nonce:    tx.Nonce(),
+				GasLimit: tx.Gas(),
+				GasPrice: tx.GasPrice().String(),
+				ChainID:  cs.ChainID(),
+				Action: &iotextypes.ActionCore_EvmTxContainer{
```

Comment:
seems misleading name, It actually is generic tx container, not only for execution